app/javascript/hooks/useScrollToElement.ts

```diff
@@ -56,7 +58,9 @@ export function useScrollToElement<T extends HTMLElement = HTMLElement>(
 
       const rafId1 = requestAnimationFrame(() => {
         rafId2 = requestAnimationFrame(() => {
-          elementRef.current?.scrollIntoView(options);
+          setTimeout(() => {
+            elementRef.current?.scrollIntoView(options);
+          }, delay);
```

Comment:
Had to add a delay so that when changing taxonomy whilst being in Black Friday mode, it does not immediately scroll down to the results. This lets the user realize they are on the taxonomy's page with the appropriate design.